lib/beaker/options/hosts_file_parser.rb

```diff
@@ -85,6 +86,23 @@ def self.merge_hosts_yaml( host_options, error_message, &block )
         host_options.merge( loaded_host_options )
       end
 
+      # A helper to parse the YAML file and apply ERB templating
+      #
+      # @param [String] path Path to the file to read
+      # @param [Binding] b The binding to pass to ERB rendering
+      # @api private
+      def self.process_yaml(template, b)
+        erb_obj = if RUBY_VERSION >= '2.7'
+                     ERB.new(template, trim_mode: '-')
+                   else
+                     ERB.new(template, nil, '-')
+                   end
+        if RUBY_VERSION >= '2.6'
```

Comment:
Ruby 2.4 and 2.5 (or rather, their Psych version) do not accept permitted_classes as an argument to safe_load